Source/WebCore/css/calc/CSSCalcTree+Serialization.cpp

```diff
@@ -375,8 +375,8 @@ void serializeMathFunctionArguments(StringBuilder& builder, const IndirectNode<A
     WTF::switchOn(anchor->side,
         [&](CSSValueID valueID) {
             builder.append(nameLiteralForSerialization(valueID));
-        }, [&](double percentage) {
-            builder.append(percentage, '%');
+        }, [&](double value) {
+            builder.append(100 * value, '%');
```

Comment:
I should have caught this in the last one, but this should probably use:

    formatCSSNumberValue(builder, value * 100, CSSPrimitiveValue::unitTypeString(CSSUnitType::CSS_PERCENTAGE));

as that will handle all the formatting edge cases (inf, lots of decimal places, etc).

Reply:
I'm so glad you mentioned this! I noticed this a few hours ago looking over the last merged patch and was going to bring it up, but didn't get a chance before starting my work day.

Reply:
That's a mouthful!